ResearchKit/Consent/ORKConsentReviewStep.h

```diff
@@ -67,12 +67,30 @@ ORK_CLASS_AVAILABLE
 /**
  Returns an initialized consent review step using the specified identifier, signature, and consent document.
```

Comment:
Please include here the new requiresScrollToBottom parameter for a more complete documentation.